### PR TITLE
[NEW] API to fetch permissions & user roles

### LINKED ISSUE #9518: [NEW] Permissions REST API
# Description

Create an API that returns `permissions/get` results via REST.

### PR DESCRIPTION
**Pull request by rafaelks**
@RocketChat/core 

Closes #9518

## Progress

- [x] API: permissions
- [x] API: user.roles
- [x] Tests
- [x] [Docs](https://github.com/RocketChat/docs/pull/527)

### CODE REVIEW

**Comment by rafaelks** [comment]
Docs created: https://github.com/RocketChat/docs/pull/527.

File: packages/rocketchat-api/server/v1/permissions.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	This API returns all permissions that exists
+	on the server, with respective roles.
+
+	Method: GET
+	Route: api/v1/permissions
+ */
+RocketChat.API.v1.addRoute('permissions', { authRequired: true }, {
+	get() {
+		let result;
+		Meteor.runAsUser(this.userId, () =>
```
Comment (rodrigok) [inline packages/rocketchat-api/server/v1/permissions.js]: @MarcosSpessatto could you change this to 

```javascript
const result = Meteor.runAsUser(this.userId, () => Meteor.call('permissions/get'));
```

`Meteor.runAsUser` returns the result of the passed function.

File: packages/rocketchat-api/server/v1/users.js [hunk 2]
```diff
@@ -270,3 +270,27 @@ RocketChat.API.v1.addRoute('users.createToken', { authRequired: true }, {
...
+
+	Method: GET
+	Route: api/v1/user.roles
+ */
+RocketChat.API.v1.addRoute('user.roles', { authRequired: true }, {
+	get() {
+		let result;
+		let currentUserRoles = {};
+
+		Meteor.runAsUser(this.userId, () =>
```
Comment (rodrigok) [inline packages/rocketchat-api/server/v1/users.js]: Same here

**Comment by rafaelks** [comment]
Is this PR still missing something @rodrigok & @MarcosSpessatto?

**Pull request merged**